src/main/java/gtPlusPlus/core/item/chemistry/GenericChem.java

```diff
@@ -484,6 +491,34 @@ private void recipeCarbonDisulfide() {
 		
 	}
 
+	private void recipeMutatedLivingSolder() {
+
+		//Endgame soldering alloy meant for the bioware circuit line and beyond.
+		CORE.RA.addChemicalPlantRecipe(
+				new ItemStack[] {
+						ItemUtils.getSimpleStack(GenericChem.mInfiniteMutationCatalyst, 1),
+                        ItemList.Circuit_Chip_Biocell.get(64),
+						ItemList.Gravistar.get(8),
+						Materials.InfinityCatalyst.getDust(2)
+
+				},
+				new FluidStack[] {
+						FluidUtils.getFluidStack("plasma.tin", 18000),
+						FluidUtils.getFluidStack("plasma.bismuth", 18000),
+						FluidUtils.getFluidStack("cryotheum", 4000)
+				},
+				new ItemStack[] {
+
+				},
+				new FluidStack[] {
+						MISC_MATERIALS.MUTATED_LIVING_SOLDER.getFluidStack(40000)
+				},
+				20*800,
```

Comment:
800s at UEV for 40buckets, that's like 2.5mb/t if i didn't made any miscalculation.
I'd keep that ratio, but would divide the time by 4.

Reply:
It's 144 seconds per recipe, I mentioned this on the PR itself.

Reply:
with the awakened draconium coils it becomes 13.88 mb/t. depending on the usage of such new alloy, it may become a bottleneck again, unless you use low amount in recipes as @Sphyix said: otherwise you'll need to spam machines again, and that's boring. (Also the power consumption dedicated to this lol)

Reply:
I'll keep the recipe time and lower the energy usage by one tier, then. Should be good enough then, with the added parallel.